Update minting-nft to use transction build command (#377)
cardano-foundation/developer-portal: docs/native-tokens/minting-nfts.md

echo $script
```

If everything is set, run the following command:

```bash
cardano-cli transaction build \
--mainnet \
--alonzo-era \
--tx-in $txhash#$txix \
--tx-out $address+$output+"$tokenamount $policyid.$tokenname" \
--change-address $address \
--mint="$tokenamount $policyid.$tokenname" \
--minting-script-file $script \
--metadata-json-file metadata.json  \
--invalid-hereafter $slotnumber \
--witness-override 2 \
--out-file matx.raw
```

> If the above command generate an output similar to:

Reviewer: Reads incorrectly

Reviewer: The suggestion was only for flow of the sentence (`If ... : ` felt like there is a continuation below, all good otherwise)

```suggestion
The above command may generate output as per below:
```
